debian/kolibri-server.init

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
we should not depend on `bash`, it has been removed from `essential` several times in Debian and it will likely be removed again in the future. Theorically you should be able to remove it from your system and continue working.

Reply:
Oh okay, I just found it in several init.d script templates and thought it was safe. It was necessary to make a simple substitution pattern that `sh` didn't support. I can use `sed` instead.

Reply:
We use bash in all of the Debian install scripts for the `kolibri` package btw.

Reply:
Decided to use bash a little more consistently (see added conditional in postinst)

Reply:
I don't like this dependency (it also makes an additional change in debian/control needed). I'd prefer to keep things simpler

Reply:
This is where you mentioned adding bash as a dependency :)

Reply:
I don't know what other comment it could be . I am always against adding more dependencies/code, etc. if I told you somewhere to use bash it would be my mistake.
I remember telling in another comment that a Debian package does not need to add a bash dependency currently because it's in `essential` now, but that does not mean I want to depend on it.